### ISSUE TITLE
Resizing transparent animated gifs keeps previous frames

### ISSUE THREAD

**Issue by montanaflynn** [willnorris/imageproxy#82]
The [original gif](https://storage.googleapis.com/user-uploaded-stickers/1efee6fa-430f-4ab8-8a09-863c20dffc9e.gif) has a transparent background but when [resized](https://api.gifs.com/resize/211.2x/https://storage.googleapis.com/user-uploaded-stickers/1efee6fa-430f-4ab8-8a09-863c20dffc9e.gif) each previous frame is still visible.

I think this might be related to #80.

**Comment by montanaflynn** [comment]
Here is a fix for another animated gif bug with a similar problem (it keeps previous frames instead of changing them to transparent) on this repo:

https://github.com/jaschaephraim/pingpong/blob/master/img.go#L75-L85 https://github.com/jaschaephraim/pingpong/blob/master/util.go#L13-L25

Without transparent fix:
![3o85xug8npw9okvquc](https://cloud.githubusercontent.com/assets/24260/23480463/d7b8ab88-fe7c-11e6-8527-6149f8630ebb.gif)

With transparent fix:
![3o85xug8npw9okvquc](https://cloud.githubusercontent.com/assets/24260/23480469/de0659d6-fe7c-11e6-9c5a-4fecd2031eba.gif)

**Comment by willnorris** [comment]
Agreed that this seems related to #80.  I hadn't spent a lot of time looking at all the edge cases relating to transparent gifs, so it doesn't surprise me terribly that there are some issues.  Feel free to try sending a pull request to fix this, or I'll try to take a look when I've got some time (which likely won't be soon).
